vault/resource_github_auth_backend_test.go

```diff
@@ -90,9 +91,7 @@ func TestAccGithubAuthBackend_ns(t *testing.T) {
 			{
 				Config: testAccGithubAuthBackendConfig_ns(ns, path, testGHOrg),
 				Check: resource.ComposeTestCheckFunc(
-					testutil.TestAccCheckAuthMountExists(resourceName,
-						&resAuth,
-						testProvider.Meta().(*provider.ProviderMeta).MustGetClient()),
+					githubAuthMountExistsHelperNS(resourceName, &resAuth),
```

Comment:
This test was not getting the correct namespaced client; a wrapper method that fetches the Vault client in the correct namespace has been added, and the test now passes in CI